cobalt/build/get_build_id.py

```diff
@@ -15,20 +15,62 @@
 """Prints out the Cobalt Build ID."""
 
 import os
+import re
+import subprocess
+
 from cobalt.build.build_number import GetOrGenerateNewBuildNumber
 
+COMMIT_COUNT_BUILD_NUMBER_OFFSET = 1000000
 
-def main():
+# Matches numbers > 1000000. The pattern is basic so git log --grep is able to
+# interpret it.
+GIT_BUILD_NUMBER_PATTERN = r'[1-9]' + r'[0-9]' * 6 + r'[0-9]*'
```

Comment:
Nit: The last item here could be replaced by a `+`.

I.e. 

`GIT_BUILD_NUMBER_PATTERN = r'[1-9]' + r'[0-9]' * 6 + '+'`

Reply:
It can't, `git log --grep` doesn't support the `+` operator